ci/run_ert_ctests.sh

```diff
@@ -58,11 +66,6 @@ build_ert_clib () {
 	popd
 }
 
-build_ert_dev () {
-	enable_environment
-	pip install "${ERT_SOURCE_ROOT}[dev]"
```

Comment:
I don't immediately understand why this was removed.
Is it because `build_ert_clib` uses the build script and that implicitly sets up ert using `.[dev]` and more?

Reply:
Great. Seems like no reason to keep it. 👍